File: EASY/SUM-OF-DIGITS/sod.cpp
#include <iostream>
#include <string>
using namespace std;

class Solution {
public:
    int sumOfDigits(int n) {
        int sum = 0;
        string s = to_string(n);
        for (char c : s) {
            sum += c - '0';
        }
        return sum;
    }
};

int main() {
    Solution s;
    cout << s.sumOfDigits(1234) << endl;  // Output: 10
    return 0;
}

File: EASY/SUM-OF-DIGITS/sod.java
public class sod {
    public int sumOfDigits(int n) {
        int sum = 0;
        for (char s : Integer.toString(n).toCharArray()) {
            sum += s - '0'; // Convert char to int
        }
        return sum;
    }

    public static void main(String[] args) {
        sod s1 = new sod();
        System.out.println(s1.sumOfDigits(1234));
    }
}